setup.cfg

```diff
@@ -0,0 +1,16 @@
+[metadata]
+name = unpaddedbase64
+version = 1.1.0
+description = Unpadded Base64
+long_description = file: README.rst
```

Comment:
Oh this is much less faff than what we normally do to get the README contents.

Reply:
Yeah. Modern setuptools can be kind of OK!

Poetry puts `readme = "README.rst"` in the `pyproject.toml` and Flit does `description-file = "README.rst"`... so everyone's finally moving in this direction